katdal/visdatav4.py

```diff
@@ -116,7 +117,7 @@ def __init__(self, source, ref_ant='', time_offset=0.0, **kwargs):
         half_dump = 0.5 * self.dump_period
         self.start_time = katpoint.Timestamp(source.timestamps[0] - half_dump)
         self.end_time = katpoint.Timestamp(source.timestamps[-1] + half_dump)
-        self._time_keep = np.full(num_dumps, True)
+        self._time_keep = np.full(num_dumps, True, dtype=bool)
```

Comment:
Maybe `np.bool_` to be a bit more precise?

Reply:
Sure.